openslides/users/static/js/users/site.js

```diff
@@ -850,76 +826,117 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
 
 .controller('GroupListCtrl', [
     '$scope',
+    '$http',
+    'operator',
     'Group',
-    function($scope, Group) {
+    'permissions',
+    'gettext',
+    'Agenda',
+    'Assignment',
+    'Mediafile',
+    'Motion',
+    'User',
+    function($scope, $http, operator, Group, permissions, gettext, Agenda, Assignment, Mediafile, Motion, User) {
         Group.bindAll({}, $scope, 'groups');
+        $scope.permissions = permissions;
+
+        $scope.apps = [];
+        var lastApp = '';
+
+        // Create the main clustering with appname->permissions
+        angular.forEach(permissions, function(perm) {
+            var permissionApp = perm.value.split('.')[0]; // get appname
+            var translations = { 'agenda': Agenda.verboseName,
+                                 'assignments': Assignment.verboseName,
+                                 'core': gettext('General'),
+                                 'mediafiles': Mediafile.verboseName,
+                                 'motions': Motion.verboseName,
+                                 'users': User.verboseName };
```

Comment:
Note: We need plural for section titles (like in settings or main menu) but `Motion.verboseName` and `Assignment.verboseName` gets singular string ('Motion', 'Election') - which are also used by item-detail.html (back button).
We should define a 'verboseNamePlural' for assignments, motions, medafiles and users (agenda is singlur).